src/styles/card.scss

```diff
@@ -0,0 +1,114 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: auto;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-family: $main-font-family;
+    font-size: 12px;
+    font-weight: $medium;
+    line-height: 18px;
+    color: $primary-color;
+    align-content: space-evenly;
+    margin-bottom: 4px;
+    &:hover {
+        color: #34568b;
+      }
+  }
+
+  &__code {
+    font-family: $main-font-family;
+    font-size: 10px;
+    font-weight: $normal;
+    line-height: 14px;
+    color: $secondary-color;
+    margin-top: 4px;
+    margin-bottom: 16px;
```

Comment:
it's better to set only top o only bottom (read about margin collapse)
You've already set margin-bottom for title - no need to set margin-top: 4px here: